inscriptions/ISic000310.xml

```diff
@@ -185,60 +190,24 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="EDR" n="EDR141128" xml:lang="la">
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#TNS">
                 <ab>
-                    <lb n="1"/>
-                    <expan>
-                        <abbr>Q</abbr>
-                        <ex>uintus</ex>
-                    </expan>
-                    Lutati
-                    <supplied reason="lost">us</supplied>
-                    <gap reason="lost" extent="unknown" unit="character"/>
-                    <lb n="2"/>
-                    <expan>
-                        <abbr>q</abbr>
-                        <ex>uaestor</ex>
-                    </expan>
-                    <expan>
-                        <abbr>d</abbr>
-                        <ex>ecreto</ex>
-                    </expan>
-                    <expan>
-                        <abbr>d</abbr>
-                        <ex>ecurionum</ex>
-                    </expan>
-                    aedil
-                    <supplied reason="lost">is</supplied>
-                    <gap reason="lost" extent="unknown" unit="character"/>
-                    <lb n="3"/>pro hono
-                    <supplied reason="lost">re
-                        <choice>
-                            <sic>IIviratus</sic>
-                            <corr>duoviratus</corr>
-                        </choice>
-                    </supplied>
-                    <lb n="4"/>
-                    <supplied reason="lost">tr</supplied>iclin
-                    <gap reason="lost" extent="unknown" unit="character"/>
-                    <supplied reason="lost">- - -</supplied>
-                    <gap reason="lost" extent="unknown" unit="character"/>
-                    <supplied reason="lost"/>
-                    <lb n="5"/>t<supplied reason="lost">rib</supplied>us l
-                    <supplied reason="lost">ectis dedit</supplied>
-                    <gap reason="lost" extent="unknown" unit="character"/>
+                    <lb n="1"/><name><expan><abbr>Q</abbr><ex>uintus</ex></expan></name> <g ref="#interpunct">·</g> Lutati<supplied reason="lost">us</supplied><gap reason="lost" extent="unknown" unit="character"/>
+                    <lb n="2"/><expan><abbr>q</abbr><ex>uaestor</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>d</abbr><ex>ecreto</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>d</abbr><ex>ecurionum</ex></expan> aedil<supplied reason="lost">is</supplied><gap reason="lost" extent="unknown" unit="character"/>
```

Comment:
on the q of quaestor, add `<hi rend="supraline">q</hi>`, as per the edition in CIL (supraline is a form of abbreviation marker)